snowshu/adapters/target_adapters/postgres_adapter/postgres_adapter.py

```diff
@@ -87,8 +86,10 @@ def __init__(self, replica_metadata: dict, **kwargs):
                                      f'-h {self._credentials.host} '
                                      f'-U {self._credentials.user} '
                                      f'-d {self._credentials.database}')
-        self.DOCKER_SHARE_REPLICA_DATA = f"cp -af $PGDATA/* {self.DOCKER_REPLICA_MOUNT_FOLDER}"  # noqa pylint: disable=invalid-name
-        self.DOCKER_IMPORT_REPLICA_DATA_FROM_SHARE = f"cp -R -f {self.DOCKER_REPLICA_MOUNT_FOLDER}/* $PGDATA/"  # noqa pylint: disable=invalid-name
+        self.DOCKER_SHARE_REPLICA_DATA = f"pg_dumpall -c -U {self._credentials.user} -p 9999 > " \
```

Comment:
should we achieve the output?

Reply:
Don't understand the question. Achieve what?

Reply:
the output of the command